rabbitmq/assets/monitors/consumers_at_zero.json

```
@@ -0,0 +1,29 @@
+{
+    "version": 2,
+    "created_at": "2024-05-28",
+    "last_updated_at": "2024-05-28",
+    "title": "Number of consumers is 0 in RabbitMQ queue",
```

```suggestion
    "title": "RabbitMQ queue has 0 consumers",
```
Just a suggestion if this makes sense.